internal/provider/gen/architecture_data_source_gen.go

```diff
@@ -406,15 +470,17 @@ func ArchitectureDataSourceSchema(ctx context.Context) schema.Schema {
 }
 
 type ArchitectureModel struct {
-	Id                        types.String   `tfsdk:"id"`
-	Location                  types.String   `tfsdk:"location"`
-	ManagementGroups          types.List     `tfsdk:"management_groups"`
-	Name                      types.String   `tfsdk:"name"`
-	PolicyAssignmentsToModify types.Map      `tfsdk:"policy_assignments_to_modify"`
-	PolicyDefaultValues       types.Map      `tfsdk:"policy_default_values"`
-	PolicyRoleAssignments     types.Set      `tfsdk:"policy_role_assignments"`
-	RootManagementGroupId     types.String   `tfsdk:"root_management_group_id"`
-	Timeouts                  timeouts.Value `tfsdk:"timeouts"`
+	Id                                                      types.String   `tfsdk:"id"`
+	Location                                                types.String   `tfsdk:"location"`
+	ManagementGroups                                        types.List     `tfsdk:"management_groups"`
+	Name                                                    types.String   `tfsdk:"name"`
+	OverridePolicyDefinitionParameterAssignpermissionsSet   types.Set      `tfsdk:"override_policy_definition_parameter_assignpermissions_set"`
```

Comment:
Should there be an underscore between `assignpermissions`?

Reply:
Yeah, you're right - updated